tensorflow_addons/custom_ops/layers/cc/kernels/correlation_cost_op.cc

```diff
@@ -331,7 +331,7 @@ TF_CALL_float(REGISTER_CORRELATIONCOST_OP_CPU);
 #undef REGISTER_CORRELATIONCOST_OP_CPU
 
 // Register the GPU kernels.
-#ifdef GOOGLE_CUDA
+#if GOOGLE_CUDA
```

Comment:
Seems `#ifdef` looks better here?

Reply:
@WindQAQ Tzu-Wei, my fault, we should use #if bacause `if_cuda(["-DGOOGLE_CUDA=1"])` is passed